core/templates/services/playthrough-issues-backend-api.service.spec.ts

```diff
@@ -50,81 +46,70 @@ describe('PlaythroughIssuesBackendApiService', function() {
     is_valid: true
   }];
 
-  beforeEach(angular.mock.module('oppia', function($provide) {
-    var ugs = new UpgradedServices();
-    for (let [key, value] of Object.entries(ugs.getUpgradedServices())) {
-      $provide.value(key, value);
-    }
-  }));
-
-  beforeEach(angular.mock.inject(function($injector, $q) {
-    PlaythroughIssuesBackendApiService =
-      $injector.get('PlaythroughIssuesBackendApiService');
-    $httpBackend = $injector.get('$httpBackend');
-    PlaythroughIssueObjectFactory = $injector.get(
-      'PlaythroughIssueObjectFactory');
-    PlaythroughObjectFactory = $injector.get('PlaythroughObjectFactory');
-
-    CsrfService = $injector.get('CsrfTokenService');
-
-    spyOn(CsrfService, 'getTokenAsync').and.callFake(function() {
-      var deferred = $q.defer();
-      deferred.resolve('sample-csrf-token');
-      return deferred.promise;
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      imports: [HttpClientTestingModule],
     });
-  }));
+    httpTestingController = TestBed.get(HttpTestingController);
+    playthroughIssuesBackendApiService = TestBed.get(
+      PlaythroughIssuesBackendApiService);
+    playthroughIssueObjectFactory = TestBed.get(PlaythroughIssueObjectFactory);
+  });
 
-  afterEach(function() {
-    $httpBackend.verifyNoOutstandingExpectation();
-    $httpBackend.verifyNoOutstandingRequest();
+  afterEach(() => {
+    httpTestingController.verify();
   });
 
-  describe('.fetch', function() {
-    it('returns the issues data provided by the backend', function() {
-      var successHandler = jasmine.createSpy('success');
-      var failureHandler = jasmine.createSpy('failure');
-      $httpBackend.expectGET(
-        '/issuesdatahandler/7?exp_version=1'
-      ).respond(backendIssues);
+  describe('.fetch', () => {
+    it('returns the issues data provided by the backend', fakeAsync(() => {
```

Comment:
Please update these spec descriptions to use the form (always follow `it` with `'should '`):

```typescript
it('should return the thing', () => {
});
```